exercises/all-your-base/src/test/java/ConverterTest.java

```diff
@@ -0,0 +1,232 @@
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class ConverterTest {
+
+    /*
+     * See https://github.com/junit-team/junit4/wiki/Rules for information on JUnit Rules in general and
+     * ExpectedExceptions in particular.
+     */
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
+
+     @Test
+    public void singleBitOneToDecimal() {
+        final List<Integer> inputDigits = Collections.unmodifiableList(Arrays.asList(1));
```

Comment:
I for two appreciate attention to little details like making the list immutable.  (I say, "I for two..." instead of "I for one..." because I've seen @matthewmorgan also complement this kind of thing in the past.)

Thanks!